pages/docs/tutorials/getting-started/slackbot-websocket-tutorial.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Create an AsyncAPI Document for a SlackBot with WebSockets
+description: In this tutorial, you'll learn how to create an AsyncAPI document designed for a Slack application that operates in Socket Mode using the WebSockets protocol.
+weight: 80
```

Comment:
```suggestion
weight: 140
```

Hello Vaishnavi,
After moving the document to the correct folder, as suggested by Joy, I also suggest updating the weight to  140 because we want the 'streetlight' tutorial to be the last document in the folder.

Reply:
Thanks @CynthiaPeter